lib/fs/ext4/superblock.rb

```diff
@@ -142,8 +142,8 @@ class Superblock
 
     # /////////////////////////////////////////////////////////////////////////
     # // initialize
-    attr_reader :numGroups, :fsId, :stream, :numBlocks, :numInodes, :fsId, :volName
-    attr_reader :sectorSize, :blockSize, :groupDescriptorSize
```

Comment:
`groupDescriptorSize` supposes to be called in Ext4::GroupDescriptorTable. You sure it's safe to remove it?

Reply:
👍